tutorials/datacatalog-tag-history/index.md

```diff
@@ -1,363 +1,282 @@
+title: Create a Data Catalog tag history in BigQuery using Cloud Logging and Dataflow
+description: Learn how to create a historical record of metadata in real time by capturing logs and processing them with Pub/Sub and Dataflow.
 author: anantdamle
 tags: Data Catalog, BigQuery, PubSub, Dataflow
-date_published: 2020-09-30
+date_published: 2020-09-21
 ---
 
 Anant Damle | Solutions Architect | Google Cloud
 
-Historical metadata of your data-warehouse is a treasure trove to discover not just insights about changing data patterns, but also quality and user behavior. The challenge is Data Catalog keeps a single version of metadata for fast searchability.
-
 This solution is intended for data engineers, analysts and other technical people with responsibility for metadata management, data governance and related analytics.
 
-This tutorial suggests a solution to create a historical record of metadata Data Catalog tags by creating change records in real-time through capture and parsing of the [Audit Logs](https://cloud.google.com/data-catalog/docs/how-to/audit-logging) from [Cloud Logging](https://cloud.google.com/logging) and processing them in real-time by using [Pub/Sub](https://cloud.google.com/pubsub) and [Dataflow](https://cloud.google.com/dataflow) to append into a [BigQuery](https://cloud.google.com/bigquery) table for historical analysis.
-
-![Data Catalog Tag history recording solution architecture](images/data_catalog_tag_recorder_arch.svg)
+Historical metadata about your data warehouse is a treasure trove for discovering insights about changing data patterns, data quality, and user behavior. The
+challenge is that Data Catalog keeps a single version of metadata for fast searchability.
 
-- [Concepts](#concepts)
-- [Prerequisites](#prerequisites)
-- [Objectives](#objectives)
-- [Costs](#costs)
-- [Before you begin](#before-you-begin)
-- [Setting up your environment](#setup-environment)
-- [Creating resources](#creating-resources)
-  * [Create BigQuery table](#setup-bigquery)
-  * [Configure Pub/Sub topic and subscription](#configure-pubsub)
-  * [Configure Log sink](#configure-log-sink)
-  * [Create Service Accounts](#create-service-account)
-- [Deploy the Entry Tags recording pipeline](#launch-pipeline)  
-- [Manual Test](#manual-test)
-- [Limitations](#limitations)
-- [Cleaning up](#cleanup)
-  * [Delete the project](#delete-project)
-- [License](#license)
+This tutorial suggests a solution to create a historical record of metadata Data Catalog tags by creating change records in real time by capturing and parsing
+the [Audit Logs](https://cloud.google.com/data-catalog/docs/how-to/audit-logging) from [Cloud Logging](https://cloud.google.com/logging) and processing them in 
+real time by using [Pub/Sub](https://cloud.google.com/pubsub) and [Dataflow](https://cloud.google.com/dataflow) to append into a 
+[BigQuery](https://cloud.google.com/bigquery) table for historical analysis.
 
-### Concepts<a name="concepts"></a>
```

Comment:
have the <a name="..."></a> tag will enable linking to specific sections from outside.
Should I add them again?

Reply:
No. We just author in straight Markdown for these documents, no HTML. DevSite creates anchors for these automatically when then build happens.